src/mbgl/map/transform_state.hpp

```diff
@@ -81,7 +81,7 @@ class TransformState {
 
     // Limit the amount of zooming possible on the map.
     double min_scale = std::pow(2, 0);
-    double max_scale = std::pow(2, 20);
+    double max_scale = std::pow(2, 22);
```

Comment:
Why is this not clamped at `util::MAX_ZOOM`? (I know it wasn't before, just curious)